clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -2053,26 +2053,26 @@ inline ::std::pair<Iter1, Iter1> equal_range(_ExecutionPolicy &&policy,
 // DplExtrasVector|is_iterator
 // DPCT_DEPENDENCY_END
 // DPCT_CODE
-template <typename _ExecutionPolicy, typename key_t, typename key_out_t, 
+template <typename _ExecutionPolicy, typename key_t, typename key_out_t,
           typename OffsetIteratorT>
 inline ::std::enable_if_t<dpct::internal::is_iterator<key_t>::value &&
                           dpct::internal::is_iterator<key_out_t>::value>
 segmented_reduce_argmin(_ExecutionPolicy &&policy, key_t keys_in,
-                        key_out_t keys_out, ::std::int64_t n,
-                        ::std::int64_t nsegments, OffsetIteratorT begin_offsets,
+                        key_out_t keys_out, ::std::int64_t nsegments,
```

Comment:
Seems qualifier for `std` isn't required, can we remove extra `::`, What do you think?

Reply:
https://github.com/oneapi-src/SYCLomatic/pull/902 should resolve inconsistencies in the file.